metricbeat/module/elasticsearch/node_stats/data.go

```diff
@@ -110,6 +156,12 @@ var (
 				},
 			}),
 			"search": c.Dict("search", s.Schema{
+				"fetch_time": s.Object{
+					"ms": c.Int("fetch_time_in_millis"),
+				},
+				"fetch_total": s.Object{
```

Comment:
I'm curious if there's an advantage to keep the `fetch_` prefix in the output data.

Reply:
To be able to tell them apart from their `query_*` siblings

Reply:
Ah, yeah, duh. Too zoned into the diff.